<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sách Mini</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>571</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255)</string>
    </property>
    <property name="title">
     <string>Thông tin nhập: </string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tên khách hàng: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTenKH">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBanMoi">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>84</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Bán mới</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSoLuong">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTinh">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>84</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Tính</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Số lượng: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Khách hàng là SV: </string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSV">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>75</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonThongKe">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>120</y>
       <width>84</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Thống kê</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonThoat">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>160</y>
       <width>84</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Thoát</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditThanhTien">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Thành tiền: </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>571</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255)</string>
    </property>
    <property name="title">
     <string>Thông tin thống kê: </string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tổng số KH: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTongSoKH">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTongSV">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tổng số KH là SV: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTongDoanhThu">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Tổng doanh thu: </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditTenKH</tabstop>
  <tabstop>lineEditSoLuong</tabstop>
  <tabstop>checkBoxSV</tabstop>
  <tabstop>lineEditThanhTien</tabstop>
  <tabstop>pushButtonBanMoi</tabstop>
  <tabstop>pushButtonTinh</tabstop>
  <tabstop>pushButtonThongKe</tabstop>
  <tabstop>pushButtonThoat</tabstop>
  <tabstop>lineEditTongSoKH</tabstop>
  <tabstop>lineEditTongSV</tabstop>
  <tabstop>lineEditTongDoanhThu</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
